signer/src/transaction_signer.rs

```diff
@@ -152,7 +154,7 @@ pub struct TxSignerEventLoop<Context, Network, Rng> {
     /// verification of the Stacks rotate-keys transaction.
     pub wsts_frost_state_machines: LruCache<StateMachineId, FrostCoordinator>,
     /// Results of the FROST verification rounds.
-    pub wsts_frost_results: LruCache<StateMachineId, bool>,
+    pub wsts_frost_mock_txs: LruCache<StateMachineId, UnsignedMockTransaction>,
```

Comment:
Should we merge this with the above (ie, `LruCache<StateMachineId, (FrostCoordinator, UnsignedMockTransaction)>`, or wrapping both in a struct)? So that we are sure we always have both available, given that we have an lru and the state machine usage can be decoupled from the mock transaction we may end up  in some strange edge case

Reply:
Yeah I've been wanting to do this, just haven't gotten to it yet -- will try to get it done soon

Reply:
Addressed in a different PR